nanome/_internal/structure/io/sdf/parse.py

```diff
@@ -80,8 +83,14 @@ def parse_model(lines):
                         bond.bond_order = record_chunk_int(line, 7, 9)
                         model.bonds.append(bond)
                         bond_counter = bond_counter - 1
-                    elif line[0] == 'm':
+                    elif line[0] == 'M':
```

Comment:
would it make sense to make this `line[0].upper()`?

Reply:
nah, SDF standard is capital M